gerrymandering/gerrymandering.ipynb

```diff
@@ -97,9 +102,51 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "df = dd.io.parquet.read_parquet('./data/census.snappy.parq', engine='pyarrow')\n",
-    "df = df.persist()\n",
-    "census_points = hv.Points(df, kdims=['easting', 'northing'], vdims=['race'])"
+    "df = dd.read_parquet('./data/census.snappy.parq', engine='fastparquet')\n",
+    "# Load just a fraction of the points if your computer doesn't have enough memory\n",
+    "# df = df.sample(frac=0.01)\n",
+    "df = df.persist()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# census_points = hv.Points(df, kdims=['easting', 'northing'], vdims=['race'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x_range, y_range = ((-13884029.0, -7453303.5), (2818291.5, 6335972.0)) # Continental USA"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "shaded = df.hvplot.points(\n",
+    "    'easting', 'northing',\n",
+    "    datashade=True,\n",
+    "    # rasterize=True,\n",
+    "    aggregator=ds.count_cat('race'),  # color='race': I think this should work instead of aggregator=... but I got an error.\n",
+    "    color_key=color_key,  # or cmap=color_key, same outcome apparently\n",
```

Comment:
I'd suggest consistently using color_key rather than cmap when plotting categorical data, even if cmap is accepted.

Reply:
Ok!